src/commands/staff/AddOrChangeName.ts

```diff
@@ -0,0 +1,363 @@
+import {ArgumentType, BaseCommand, ICommandContext, ICommandInfo} from "../BaseCommand";
```

Comment:
AddOrChangeName

1. Crashes when I attempt to edit one of my nicknames in the database.  Crash occurs once a name is selected in the dropdown.  It could be a result of my perms (bot developer role), since it works on the test user provided.  Details of crash below.
![image](https://user-images.githubusercontent.com/97406520/149032726-dccee5bd-c258-4992-80b5-7debc6384244.png)
![image](https://user-images.githubusercontent.com/97406520/149032746-1a8af297-71b4-49fa-a69b-a74e8db86a51.png)

2. Consider rewording "Edit" to "Replace" since that is what ends up happening when we edit a name.
"Do you want to add NAME or replace one of dmansdmans names with NAME"
Add, Replace, Cancel
![image](https://user-images.githubusercontent.com/97406520/149032884-2edf6476-02cc-48f3-88ad-b8845b45c391.png)

Reply:
Are you sure you want to replace one of  @!TestT  | nameThree | nameFour's names with NEWNAME? Select an IGN to replace with the new IGN.  Otherwise, press the Cancel button
![image](https://user-images.githubusercontent.com/97406520/149033349-bd49d5b4-f740-48e3-a263-8536200f191a.png)

Reply:
BUG: Name was not replaced: 
![image](https://user-images.githubusercontent.com/97406520/149033436-3bf09265-a47d-46ba-b02c-be67b07b4a8c.png)

Reply:
Weird behavior where I could change Beep's nickname once, but could not change it again.  If intended, needs clarification for user.
![image](https://user-images.githubusercontent.com/97406520/149034064-c61e5b7f-072f-42d3-8b6b-7f95a623d4d1.png)

Reply:
I removed all nicknames.  Then I added "GoodBeep" but it was "already in database."  Then I removed "GoodBeep" and it wasn't there.  I think these commands have lots of weird interactions to resolve.

![image](https://user-images.githubusercontent.com/97406520/149034342-b318a852-1d00-4a64-b2ba-1686d5b8830f.png)